CONTRIBUTING.rst

```diff
@@ -315,13 +315,13 @@ Here are the key steps you need to go through to copy the repo before contributi
 
 3. install the forked version of `nilearn`::
 
-      pip install . -e
+      pip install -e .[dev]
```

Comment:
```suggestion
      pip install -e '.[dev]'
```
(prevent shell from interpreting the square brackets as a glob pattern)

Reply:
@jeromedockes based on reports from `tedana` Windows users, using single quotes there seems to cause problems. For context, the relevant issue is https://github.com/ME-ICA/tedana/issues/844.

Is it possible that these instructions will cause problems for Windows users?

Reply:
how about `pip install -e ".[dev]"`?

Reply:
The user only tried single quotes, so I don't know. Do any of the devs use Windows?